pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/elastic/search/ElasticSearchParticipantDto.java

```diff
@@ -114,6 +123,11 @@ public List<String> getGovernedUsers() {
         return governedUsers == null ? Collections.emptyList() : governedUsers;
     }
 
+
+    /**
+     * Returns the participant id by checking the profile
+     * for guid and altpid, preferentially returning the guid.
+     */
```

Comment:
Initially I had changed `getParticipantId()` so that it would return the `queriedParticipantId` when the profile was null or the guids were empty, but this felt pretty clunky and overly complex to me, so I opted for a separate field whose purpose is more clear.